src/play_controller.cpp

```diff
@@ -458,6 +458,7 @@ void play_controller::maybe_do_init_side()
 
 void play_controller::do_init_side()
 {
+	{
```

Comment:
Putting sync in a std::unique_ptr might be nicer than having the entire function's indentation off-by-one.